ons-spark/ancillary-topics/pandas-udfs.ipynb

```diff
@@ -314,92 +333,99 @@
-       "    </tr>\n",
-       "    <tr>\n",
-       "      <th>8</th>\n",
-       "      <td>4306091</td>\n",
-       "      <td>4.25</td>\n",
-       "    </tr>\n",
-       "    <tr>\n",
-       "      <th>9</th>\n",
-       "      <td>4715091</td>\n",
-       "      <td>4.25</td>\n",
+       "      <td>Rabbit</td>\n",
+       "      <td>554650</td>\n",
+       "      <td>192350</td>\n",
+       "      <td>532066</td>\n",
+       "      <td>180010</td>\n",
+       "      <td>25.735436</td>\n",
        "    </tr>\n",
        "  </tbody>\n",
        "</table>\n",
        "</div>"
       ],
       "text/plain": [
-       "  incident_number  minute_cost(incident_duration, incident_cost)\n",
-       "0          139091                                           4.25\n",
-       "1          275091                                           4.25\n",
-       "2         2075091                                           4.25\n",
-       "3         2872091                                           4.25\n",
-       "4         3553091                                           4.25\n",
-       "5         3742091                                           4.25\n",
-       "6         4011091                                           4.25\n",
-       "7         4211091                                           4.25\n",
-       "8         4306091                                           4.25\n",
-       "9         4715091                                           4.25"
+       "  incident_number animal_group  easting_rounded  northing_rounded  \\\n",
+       "0          139091          Dog           532350            170050   \n",
+       "1          275091          Fox           534750            167550   \n",
+       "2         2075091          Dog           528050            164950   \n",
+       "3         2872091        Horse           504650            190650   \n",
+       "4         3553091       Rabbit           554650            192350   \n",
+       "\n",
+       "   lfb_easting  lfb_northing   distance  \n",
+       "0       532066        180010   9.964048  \n",
+       "1       532066        180010  12.745802  \n",
+       "2       532066        180010  15.586271  \n",
+       "3       532066        180010  29.408275  \n",
+       "4       532066        180010  25.735436  "
       ]
      },
-     "execution_count": 28,
+     "execution_count": 9,
      "metadata": {},
      "output_type": "execute_result"
     }
    ],
    "source": [
     "@pandas_udf(T.DoubleType(), F.PandasUDFType.SCALAR)\n",
-    "def minute_cost(duration:pd.Series, incidentcost:pd.Series) -> pd.Series:\n",
-    "    duration_minutes = duration*60\n",
-    "    return incidentcost/duration_minutes\n",
-    "\n",
-    "incident_minute_cost = rescue.select('incident_number', minute_cost('incident_duration', 'incident_cost'))\n",
+    "def distx(easting_rounded:pd.Series, lfb_easting:pd.Series, northing_rounded:pd.Series, lfb_northing:pd.Series)->pd.Series:\n",
```

Comment:
I think it would be a good idea to have a more informative name for the function, something like "distance_from_nearest_lfb" (bonus points if you can think of something a bit more concise!). 

Also, am I right in thinking the function is essentially using Pythagoras' theorem to calculate the distance, and easting/northing are essentially "points" as to how far north or east something is? That was a little unclear to me from the outset so might be a good idea to just explain slightly about how the function works (I think it seems to make sense as long as my assumptions are correct - but would be good to just have a couple of lines explaining it if so)